wardrobe-auth/data/token.go

```diff
@@ -0,0 +1,6 @@
+package data
+
+type Tokens struct {
+	JWTToken     string `json:"jwt_token"`
+	RefreshToken string `json:"refresh_token_id"`
```

Comment:
Что за id? Я бы не добавлял это в контракт
```suggestion
	RefreshToken string `json:"refresh_token"`
```